wadas/ui/access_request_dialog.py

```diff
@@ -64,6 +67,34 @@ def to_json(self):
         with open(MODEL_REQUEST_CFG, 'w') as f:
             json.dump(data, f, indent=4)
 
+class TermsAndConditionsDialog(QDialog):
+    """Class to represent TERMS_AND_CONDITIONS_OF_USE file in a QDialog"""
+    def __init__(self, terms_accepted=False):
+        super().__init__()
+        # Acceptance status
+        self.terms_accepted = terms_accepted
+        # Don't show again Terms and Conditions (if accepted)
+        self.dont_show = False
+
+        # UI
+        self.setWindowTitle("WADAS Terms and Conditions of use")
+        self.setGeometry(150, 150, 500, 400)
+        self.setWindowIcon(QIcon(os.path.join(module_dir_path, "..", "img", "mainwindow_icon.jpg")))
```

Comment:
You might consider to use pathlib.Path, by converting module_dir_path to this class.